### PR TITLE
updated autoinstrument as a single word

### PR DESCRIPTION
**Pull request by Rajamanosankari**

### CODE REVIEW

**Comment by chalin** [comment]
Note to @open-telemetry/docs-approvers - we have a PR open to handle this check (and auto-fix). Maybe we'd better handle all cases together, or maybe this PR can go in first. I'll let y'all review and decide.

**Review by theletterf — APPROVED** [review]

**Comment by theletterf** [comment]
Let's move this one in - no biggie!

**Review by chalin — APPROVED** [review]
@theletterf, ok, LGTM then.

**Pull request merged**
